src/helper-functions/urls.js

```diff
@@ -43,6 +43,9 @@ const getActiveTasksURL = (rdsId) => `${baseURL}/tasks/${rdsId}?status=active`;
 const getAddMemberRoleURL = (rdsId) =>
   `${baseURL}/members/moveToMembers/${rdsId}`;
 
+const getArchiveMemberUrl = (rdsId) =>
```

Comment:
Why is it URL above, but Url here?

Reply:
Yeah, it should be a `URL `to keep code consistent.